dev/plugins/hu.elte.txtuml.export.javascript/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,32 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: txtUML Javascript Visualizer
+Bundle-SymbolicName: hu.elte.txtuml.export.javascript;singleton:=true
+Bundle-Version: 0.5.1.qualifier
+Bundle-Activator: hu.elte.txtuml.export.javascript.Activator
+Bundle-Vendor: ELTE
+Require-Bundle: 
+ hu.elte.txtuml.layout.visualizer,
+ hu.elte.txtuml.layout.export,
+ hu.elte.txtuml.api.model,
+ hu.elte.txtuml.api.layout,
+ hu.elte.txtuml.utils,
+ hu.elte.txtuml.export.uml2,
+ org.eclipse.debug.ui,
+ org.eclipse.jdt.core,
+ org.eclipse.jdt.launching,
+ org.eclipse.ui.ide,
+ org.eclipse.uml2.uml,
+ org.eclipse.emf.ecore,
+ org.eclipse.jdt.ui,
+ com.google.guava,
+ hu.elte.txtuml.utils.eclipse,
+ org.eclipse.ui.workbench,
+ org.eclipse.jface,
+ org.eclipse.osgi,
+ org.eclipse.core.runtime,
+ hu.elte.txtuml.export.papyrus,
+ org.eclipse.persistence.moxy,
+ org.eclipse.papyrus.uml.diagram.common
```

Comment:
The dependency includes:
*.elementsarrangers.txtumllayout.LayoutVisualizerManager
*.elementsarrangers.ArrangeException
*.layout.txtuml.TxtUMLLayoutDescriptor
*.wizardz.VisualizeTxtUMLPage
and
*.preferences.PreferencesManager

All of the above mentioned classes can be refactored not to be part of the papyrus component. However we will have to discuss the details.